src/SonarQube.Bootstrapper/Program.cs

```diff
@@ -33,6 +34,11 @@ public static int Main(string[] args)
         {
             var logger = new ConsoleLogger(includeTimestamp: false);
             Utilities.LogAssemblyVersion(logger, Resources.AssemblyDescription);
+#if IS_NET_FRAMEWORK
+            logger.LogInfo("Using the .NET Framework version of the Scanner for MSBuild");
```

Comment:
I tweaked the wording very slightly - used to say "_Executing_ the XXX _build_ of..."